From 74c383e7da3e20c201e3b0c847a3d2f6fa0dbdc6 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Sun, 29 Sep 2019 14:48:28 +0800
Subject: [PATCH] DRM: mhdp: HDMI: skip scdc write return check

HDMI sink that only support HDMI1.4 is not support
SCDC read/write. Skip the SCDC write check and continue
HDMI initialize process.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
---
 drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

--- a/drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c
+++ b/drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c
@@ -173,10 +173,8 @@ void cdns_hdmi_mode_set(struct cdns_mhdp
 	int ret;
 
 	ret = hdmi_sink_config(mhdp);
-	if (ret < 0) {
-		DRM_ERROR("%s failed\n", __func__);
-		return;
-	}
+	if (ret < 0)
+		DRM_DEBUG("%s failed\n", __func__);
 
 	ret = cdns_hdmi_ctrl_init(mhdp, mhdp->hdmi.hdmi_type, mhdp->hdmi.char_rate);
 	if (ret < 0) {
